tmt/steps/execute/internal.py

```diff
@@ -146,7 +146,9 @@ def _test_environment(
         assert self.parent is not None
         assert isinstance(self.parent, tmt.steps.execute.Execute)
 
+        environment["TMT_TEST_NAME"] = test.name
         environment["TMT_TEST_DATA"] = str(data_directory / tmt.steps.execute.TEST_DATA)
+        environment["TMT_TEST_METADATA"] = os.path.join(data_directory, test.metadata_filename)
```

Comment:
We recently switched from `os.path` to using `pathlib.Path`, therefore this should read `str(data_directory / test.metadata_filename)`.

Reply:
I tried that, but both `data_directory` and `metadata_filename` were strings, so the `/` operator didn't work.
Then I saw `os.path` used just 1-2 lines after this code, so I used the same approach.

Should I really convert `data_directory` to pathlib instead?

Reply:
Hmm, that's weird... `data_directory = self.data_path(test, full=True, create=True)`, `data_path(...) -> Path`, hence `data_directory` should already be a `Path` instance. Or I'm missing something..

Reply:
Ah, I see, this was a *very* recent change.

Reply:
It must be a `Path` instance, just one line above, it's used as a `Path` instance, `environment["TMT_TEST_DATA"] = str(data_directory / tmt.steps.execute.TEST_DATA)`. So it must have been some fluke, or, I guess it might have been an outdated branch, as there's no `os.path` anywhere in this file anymore.

Reply:
Two weeks ago, https://github.com/teemtee/tmt/pull/1776#event-8561597734, plus a few follow-up patches.